ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -2230,6 +2230,11 @@ AHU_BSPC_CHWSC_ECON_EFSS_EFVSC_HWSC_PHWSC_SFSS_SFVSC_SSPC:
   - SFVSC
   - SSPC
 
+AHU_UNDEFINED:
+  description: "An AHU with no telemetry data."
```

Comment:
Note in description that this should only be used in special cases where AHU information is not available.

Reply:
added